unison-cli/src/Unison/CommandLine/FZFResolvers.hs

```diff
@@ -9,6 +9,15 @@ module Unison.CommandLine.FZFResolvers
     projectBranchOptionsWithinCurrentProject,
     fuzzySelectFromList,
     multiResolver,
+    definitionResolver,
```

Comment:
Now that we don't need custom arg descriptions in each resolver, we can just export the whole resolver type :)